src/scrabble.js

```diff
@@ -1,5 +1,49 @@
-function scrabble() {
-
+function scrabble(string) {
+  if (string === null) {
+    return 0
+  } else {
+    const word = string.toUpperCase()
```

Comment:
Like that you transformed the string to Uppercase, find it easier to identify uppercase letters.